NGS2_WITNESS_Cycle1_MASTER.R

```diff
@@ -0,0 +1,59 @@
+#Created by Pablo Diego Rosell, PhD, for Gallup inc. on August 2017
+
+### System information
+
+a=Sys.info()
+print(a)
+b=sessionInfo(package = NULL)
+print(a)
+
+### Set base directory
+
+setwd(X:/DARPA_NGS2/CONSULTING/Analytics/Cycle1)
+rm(list=ls())
+
+### Load all required packages
+
+if (!require("pacman")) install.packages("pacman")
+library ("pacman")
+pacman::p_load(multiwayvcov, lmtest, dplyr, reshape2, multiwayvcov, lmtest, Hmisc, corrgram, PerformanceAnalytics, 
+               doBy, car, ggplot2, DT, utils, lme4, rstan, rstanarm)
+
+### Process empanelment data
+
+source ("NGS2_WITNESS_empanelment_prep.R")
+
+### Process breadboard data
+
+source ("NGS2_WITNESS_Cycle1_experiment_prep.R")
+
+### Process coloring game data
+
+source ("NGS2_WITNESS_Cycle1_coloring_prep.R") ### TO BE ADDED (Pablo)
+
+### Response funnel 
+
+source ("NGS2_WITNESS_Cycle1_response.R") ### TO BE ADDED (Matt)

```

Comment:
These `### TO BE ADDED` do not exist yet, correct? I would suggest that we at least comment these lines that don't exist yet to ensure a clean run if someone was to execute this. In addition, authors should be aware of the script names so we don't have a conflict down the road (i.e., I call the script something different than what is here right now).